demo/index.js

```diff
@@ -2,6 +2,7 @@ import React from 'react';
 import ReactDOM from 'react-dom';
```

Comment:
I would love to keep the demo as simple as possible (with just the events).
Do you mind reverting these changes?

Reply:
Sure!